controller/src/beerocks/master/tasks/bml_task.h

```diff
@@ -67,6 +67,12 @@ class bml_task : public task {
         uint8_t cac_completed;
     };
 
+    struct operation_status_update_event {
+        uint8_t backhaul_connection_status;
```

Comment:
If you get the event - than why do you need backhaul connection status? Its obviously working...

Reply:
good point